packages/flutter/lib/src/rendering/object.dart

```diff
@@ -444,6 +444,23 @@ class PaintingContext extends ClipContext {
     }
   }
 
+  /// Blend further painting with a color filter.
+  /// * `offset` is the offset from the origin of the canvas' coordinate system
```

Comment:
Need an extra line here

```suggestion
  ///
  /// * `offset` is the offset from the origin of the canvas' coordinate system
```